### PR TITLE
Add story book

### PR DESCRIPTION
**Pull request by Luchanso**
I added [storybook](https://storybook.js.org/) for faster development

And add BurgerButton & ButtonGarden stories

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,9 +52,13 @@
     "seamless-immutable": "^7.1.2",
     "styled-components": "^2.2.0",
     "uuid": "^3.1.0",
-    "wordcloud": "^1.0.6"
+    "wordcloud": "^1.0.6",
+    "@storybook/react": "^3.2.12",
```
Comment (howardchung) [inline package.json]: should be in devdependencies

File: package.json [hunk 2]
```diff
@@ -50,9 +52,13 @@
...
+    "@storybook/addon-actions": "^3.2.12",
+    "@storybook/addon-links": "^3.2.12"
   },
   "devDependencies": {
+    "babel-preset-es2015": "^6.24.1",
```
Comment (howardchung) [inline package.json]: don't think this is needed?

Comment (Luchanso) [reply package.json]: Yeah, because without it not work storybook

File: package.json (hunk 1 above)
Comment (Luchanso) [reply package.json]: image

Comment (howardchung) [reply package.json]: OK, although it's a bit confusing since these dependencies don't end up in the webpack bundle

**Comment by howardchung** [comment]
OK to merge or do you want to add more stories?

**Comment by Luchanso** [comment]
I can add more stories later, in other PR, okey?

File: package.json (hunk 1 above)
Comment (Luchanso) [reply package.json]: I want fix it, maybe in first time add /* eslint-disable import/no-extraneous-dependencies */ in stories files?

**Pull request merged**